CHANGES.rst

```diff
@@ -22,7 +22,11 @@ Release History
 1.0.2 (unreleased)
 ==================
 
+**Fixed**
 
+- Fixed bug #28 where search function hangs for many search terms.
```

Comment:
The issue number is linked in the brackets afterwards, it shouldn't be mentioned in the changelog item as they should be self-contained. The links afterward are for people who want to dive deeper, they shouldn't ever _have_ to go there.

Reply:
Also, #28 is an issue so should be linked to https://github.com/nengo/nengo-sphinx-theme/issues/28 and both links should be in the same set of brackets separated by commas, not in two separate parentheses.